python/pfs/utils/coordinates/CoordTransp.py

```diff
@@ -33,22 +42,31 @@ def CoordinateTransform(xyin, za, mode, inr=0., cent=np.array([[0.], [0.]])):
         Transformation mode. Available mode is "sky_pfi", "sky_pfi_hsc"
         "pfi_mcs", "pfi_mcs_wofe", "mcs_pfi"
     inr : `float`, optional
-        Instrument rotator angle in degree. Default is 0.
-    cent : `np.ndarray`, (1, 2), optional
+        Instrument rotator angle in degree. Default is 0. Note that this value
+        is calculated by this routine for sky_pfi* transformation.
```

Comment:
It can only be calculated automatically if `cent` is present, correct?

Reply:
Well,, if you present `cent`, it will be over written now... I modified the sentence.